fastcrypto/src/secp256k1/mod.rs

```diff
@@ -378,19 +354,13 @@ impl FromStr for Secp256k1KeyPair {
 
 impl Signer<Secp256k1Signature> for Secp256k1KeyPair {
     fn try_sign(&self, msg: &[u8]) -> Result<Secp256k1Signature, signature::Error> {
-        let secp = Secp256k1::signing_only();
-        #[cfg(test)]
-        let message =
-            Message::from_slice(<sha3::Keccak256 as digest::Digest>::digest(msg).as_slice())
-                .unwrap();
+        // Sha256 is used by default
```

Comment:
I guess this is because we go according to the RFC, right?

Reply:
Which rfc are you referring to? RFC 6979 doesn't specify a specific hash function but uses sha-256 for all examples. We follow "SEC 1: Elliptic Curve Cryptography", which allows one of SHA-224, SHA-256, SHA-384 and SHA-512.